esy-lib/Tarball.re

```diff
@@ -67,6 +67,31 @@ let run = cmd => {
   };
 };
 
+let fixFilePermissionsAfterUnTar = out => {
+  let umask = lnot(System.getumask());
+  Fs.traverse(
+    ~f=
+      (p, s) => {
+        let p = EsyBash.normalizePathForCygwin(Path.show(p));
+        try%lwt(
+          switch (s.st_kind) {
+          | Unix.S_LNK
+          | Unix.S_REG =>
+            let%lwt () = Lwt_unix.chmod(p, s.st_perm lor 0o644 land umask);
+            Lwt.return(Run.return());
+          | Unix.S_DIR =>
+            let%lwt () = Lwt_unix.chmod(p, s.st_perm lor 0o755 land umask);
+            Lwt.return(Run.return());
```

Comment:
Nit: here and other places. Let's use `RunAsync.return()`

https://github.com/esy/esy/blob/master/esy-lib/RunAsync.re#L3

Reply:
Done